data/json/monster_factions.json

```
@@ -493,5 +493,10 @@
     "name": "sewer_snake",
     "base_faction": "mutant_piscivores",
     "neutral": "sewer_snake"
+  },
+  {
+    "type": "MONSTER_FACTION",
+    "name": "dark_wyrm",
+    "neutral": "mon_dark_wyrm"
```

```suggestion
    "name": "dark_wyrm",
    "base_faction": "mutant",
    "neutral": "dark_wyrm"
```
The base faction helps make sure they have the proper reactions to other things and vice-versa.